### PR TITLE
update links to ppp and avatars on about us page - was previously bro…

### PR DESCRIPTION
**Pull request by ningsongshen**
…ken as can be tested here: https://ay2324s2-cs2113-t12-1.github.io/tp/AboutUs.html

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
...
-| ![](https://avatars.githubusercontent.com/u/90093905?v=4) | Benjamin Goh Guan Rui |   [Github](https://github.com/BenjoBurger)   |   [Portfolio](docs/team/johndoe.md)    |
-| ![](https://via.placeholder.com/100.png?text=Photo) |     Ahmed Shaiyan     |  [Github](https://github.com/AhmedShaiyan)   |   [Portfolio](docs/team/johndoe.md)    |
+| ![](https://avatars.githubusercontent.com/u/12574693?s=400&v=4) |     Ningsong Shen     |  [Github](https://github.com/ningsongshen)   |  [Portfolio](team/ningsongshen.html)   |
+| ![](https://avatars.githubusercontent.com/u/110774636?s=96&v=4) |  Anne Leong Song En   |    [Github](https://github.com/anneleong)    |    [Portfolio](team/anneleong.html)    |
+| ![](https://avatars.githubusercontent.com/u/110652271?s=96&v=4) |   Acharya Hiteshri    | [Github](https://github.com/HiteshriAcharya) | [Portfolio](team/hiteshriacharya.html) |
+| ![](https://avatars.githubusercontent.com/u/90093905?v=4) | Benjamin Goh Guan Rui |   [Github](https://github.com/BenjoBurger)   |   [Portfolio](team/benjoburger.html)   |
```
Comment (BenjoBurger) [inline docs/AboutUs.md:9]: this does not look like benjamin, pls remove!

**Review by BenjoBurger — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (anneleong) [reply docs/AboutUs.md:9]: Looks good to me, good work ningsong :)

**Review by anneleong — APPROVED** [review]
Lgtm ;)

**Review by anneleong — APPROVED** [review]
lgtm :)

**Pull request merged**